elixir/deploy

```diff
@@ -4,10 +4,28 @@
 # Use of this source code is governed by a BSD-style
 # license that can be found in the LICENSE file.
 
+REPO=${REPO:-https://packages.erlang-solutions.com}
+
+if [[ $ERLANG_VERSION ]] ; then
+	ERLANG_TARGET="$REPO/erlang/esl-erlang/FLAVOUR_1_general/esl-erlang_${ERLANG_VERSION}-1~ubuntu~trusty_amd64.deb"
+
+	if ! (curl -s --retry 3 -L $ERLANG_TARGET -o /tmp/erlang.deb); then
```

Comment:
I suggest calling curl with `-fsS`.

`-f` will make curl return a status code > 0 if the return code is != 200;
`-S` will make curl output the error message in case of failure (otherwise suppressed by `-s`)

The same is valid for the curl call below downloading elixir.

Reply:
ok